docs/docs/tour/1-my-first-asset.md

```diff
@@ -75,7 +75,7 @@ curl --location --request PATCH 'http://localhost:8080/v1beta1/assets' \
 ```
 
 There are a few things to notice here:
-1. The HTTP method used is `PATCH`. This is because Compass does not have a dedicated `Create` or `Update` apis, it uses a single API to `Patch / Create` an asset instead. So when updating or patching your asset, you can use the same API.
+1. The HTTP method used is `PATCH`. This is because Compass does not have a dedicated `Create` apis, it uses a single API to `Patch / Create` an asset instead. So when updating or patching your asset, you can use the same API.
```

Comment:
```suggestion
1. The HTTP method used is `PATCH`. This is because Compass does not have a dedicated `Create` API, it uses a single API to `Patch / Create` an asset instead. So when updating or patching your asset, you can use the same API.
```